cfme/intelligence/reports/widgets/__init__.py

```diff
@@ -123,7 +123,7 @@ def create(self, widget_class, *args, **kwargs):
         view = navigate_to(dashboard_widget, "Add")
         view.fill_with(dashboard_widget.fill_dict, on_change=view.add_button.click)
         view = dashboard_widget.create_view(AllDashboardWidgetsView)
-        assert view.is_displayed
+        wait_for(lambda: view.is_displayed, fail_condition=False, timeout=10)
```

Comment:
You might want to use `view.wait_displayed()` instead of `wait_for`.

Reply:
No. The assert was failing because some condition in is_displayed was not met, but when waited-for, it passes after some time.

Reply:
You are probably right. I though you meant `view.is_displayed.